rllib/algorithms/alpha_zero/alpha_zero.py

```diff
@@ -344,9 +352,19 @@ def training_step(self) -> ResultDict:
                 self.local_replay_buffer.add(batch)
 
         if self.local_replay_buffer is not None:
-            train_batch = self.local_replay_buffer.sample(
-                self.config["train_batch_size"]
-            )
+            # Update target network every `target_network_update_freq` sample steps.
+            cur_ts = self._counters[
```

Comment:
See the problem? I had to read this code to understand that it can actually be both and it really depends on the _agent_step_by parameter which one will be used.